### ISSUE TITLE
Problem creating array parameter type in model RDL

### ISSUE THREAD

**Issue by MartinFijneman** [RHEAGROUP/CDP4-IME-Community-Edition#41]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the CDP4-IME
- [x] I have searched [open](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

Testing on dev-server, not possible to create array parameter type within the modelRDL of testing model.
Tried with array {2,2,2}, components using all the same parameter type "acceleration"

```
The CDP4 Services replied with code Forbidden: Forbidden: "exception:Container update of item:\u0027ParameterTypeComponent\u0027 with iid:\u0027ae9841c4-1955-49a8-8d49-0f2773b5ec77\u0027 is missing from the operation"
```

### Steps to Reproduce

With an engineering model open, within the parameter types browser:
* create  array parameter
** provide name, shortname
** choose to create in modelRDL as container
** select dimensions
** choose parameter types for components

Trying to create (once complete) not possible, gives error messag given above.

Reproducible issue

### Logs

  - [ ] There are no logs regarding the issue
  - { ] I have attached a log file or included the relevant section in this issue  
  
### System Configuration

- CDP4 version:
  - [X] CDP4-IME: 1.0.15
  - [ ] Other: 
  -
- Web Services:
  - [ ] CDP4 Web Services version: 
  - [ ] OCDT WSP version: 
- Environment (Operating system, version and so on):
  - [ ] Microsoft Excel (32/64 bit): 
  - [ ] Windows OS (32/64 bit): 
  - [ ] Additional information:

**Comment by samatrhea** [comment]
this is an issue on the server, see https://github.com/RHEAGROUP/CDP4-WebServices-Community-Edition/issues/22

**Comment by samatrhea** [comment]
fixed in CDP4-Webservices version 1.0.10
